### ISSUE TITLE
If constraint `firstItem` `superview` does not exist

### ISSUE THREAD

**Issue by markst** [Papercloud/KeyboardConstraint#5]

**Comment by markst** [comment]
@ezescaruli has this been an issue for you at any time?

**Comment by ezescaruli** [comment]
@markst

Nope actually, but I figure it could happen if the items are reversed, right? In case `firstItem` ends up being the view controller's root `view`.

Do you think it might be worth checking the `secondItem`'s `superview` as well, to consider that scenario? Like:

```
UIView *superview = [(UIView *) self.firstItem superview];

if (superview == nil) {
    superview = [(UIView *) self.secondItem superview];
}

if (superview) {
    ...
}
```

**Comment by markst** [comment]
@ezescaruli this is true. But then I think this would be the incorrect usage of the constraint, since the value applied to the `constant` then need to be reversed also?

**Comment by ezescaruli** [comment]
@markst Fair point. Let's go with your fix then.
